docs/source/conf-datasources.rst

```diff
@@ -1,22 +1,27 @@
-Datasources
-===========
+Datasource Configuration
+========================
+
+For background on supported datasource types, read `datasources <datasources.html>`_.
 
 Overview
 --------
 
-Datasources are defined in ``conf/sources.json``
+* Datasources defined in ``conf/sources.json`` control which datasources can send to and be analyzed by StreamAlert.
+* Each datasource (``kinesis``, ``s3``) contains a mapping of specific entity names (kinesis streams, s3 buckets) along with a list of logs coming from that source.
```

Comment:
"contains a mapping of specific entity names (kinesis streams, s3 buckets)" - this may be confusing for folks, what's an "entity" or "entity name"